react/week1/index.js

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import ReactDOM from "react-dom";
+
+const root = document.getElementById("root");
+
+function Header() {
+  return <h1>This is my app</h1>;
+};
+
+class TodoList extends React.Component {
```

Comment:
I think I would reconsider the naming for this component. A list is usually a collection of items, however it appears this component only renders a single todo. I would rename this `TodoListItem`